### PR TITLE
Fix RectilinearToPixel for multi-devices situations

### PR DESCRIPTION
**Pull request by RodolpheHoudas-UL**
## Summary

This merge request fixes the RectilinearToPixel function in multi-devices situations. LeapC.cs had an outdated signature and there were no implementation of RectilinearToPixelEx. The changes in these commits have been tested and confirmed to work but might require more testing to ensure that it works in all situations.

**It is highly recommended to implement PixelToRectilinearEx before merging this branch**

## Contributor Tasks

- [ ] Pair review with a member of the QA team.
- [ ] Add any release testing considerations to the MR for the next release.
- [ ] Check any relevant CHANGELOG files have been updated.
- [ ] Ensure documentation requirements are met e.g., public API is commented.
- [ ] Consider any licensing/other legal implications for this MR e.g. notices required by any new libraries.
- [ ] Add any relevant labels such as `breaking` to this MR.
- [ ] If this MR closes a Jira issue, make sure the fix version on the JIRA issue is set to the correct one.

## Reviewer Tasks

[Use emojis in review threads to communicate intent and help contributors.](https://github.com/ultraleap/UnityPlugin/blob/develop/CONTRIBUTING.md#review-threads)

- [ ] Code reviewed.
- [ ] Non-code assets e.g. Unity assets/scenes reviewed.
- [ ] Documentation has been reviewed. Includes checking documentation requirements are met and not missing e.g., public API is commented.
- [ ] Checked and agree with release testing considerations added to MR for the next release.

## Closes JIRA Issue

`Closes UNITY-857`

### CODE REVIEW

File: Packages/Tracking/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "com.ultraleap.tracking",
-  "version": "5.13.0",
+  "version": "5.13.1",
```
Comment (MattGrayUL) [inline Packages/Tracking/package.json:3]: This doesn't need to be iterated until the time of release

File: Packages/Tracking/Core/Runtime/Scripts/LeapServiceProvider.cs [hunk 2]
```diff
@@ -964,8 +964,7 @@ protected void createController()
             }
 
             string serialNumber = _multipleDeviceMode != MultipleDeviceMode.Disabled ? SpecificSerialNumber : "";
-
-            _leapController = new Controller(serialNumber.GetHashCode(), _serverNameSpace, _multipleDeviceMode != MultipleDeviceMode.Disabled);
+            _leapController = new Controller(0, _serverNameSpace, _multipleDeviceMode != MultipleDeviceMode.Disabled);
```
Comment (MattGrayUL) [inline Packages/Tracking/Core/Runtime/Scripts/LeapServiceProvider.cs:967]: this should remain serialNumber.GetHashCode() as it has been fixed by the above line on develop :)

File: Packages/Tracking/Core/Runtime/Plugins/LeapCSharp/Connection.cs [hunk 3]
```diff
@@ -38,11 +38,15 @@ public static Connection GetConnection(int connectionId = 0)
...
-            if (!Connection.connectionDictionary.TryGetValue(connectionKey, out conn))
+            if (!Connection.connectionDictionary.ContainsKey(connectionKey))
             {
                 conn = new Connection(connectionKey);
                 connectionDictionary.Add(connectionKey, conn);
             }
+            else
+            {
+                Connection.connectionDictionary.TryGetValue(connectionKey, out conn);
+            }
```
Comment (MattGrayUL) [inline Packages/Tracking/Core/Runtime/Plugins/LeapCSharp/Connection.cs:49]: This change is not necessary for the fix to work :)

**Comment by RodolpheHoudas-UL** [comment]
I have fixed my code according to your suggestion. As stated in the request, I have only implemented RectilinearToPixel for multi-devices, as I only needed to fix this one. I would highly recommend implementing PixelToRectilinear for multi-devices for features parity.

**Pull request merged**